readme.txt

```diff
@@ -0,0 +1,237 @@
+
+= Customize your newsletter’s appearance =
+
+You can customize your newsletters further by changing the header and body fonts, or picking a background color.
+
+= Newspack Blocks =
+
+Newspack Newsletters comes bundled with the Newspack Post Inserter block. This block lets you insert excerpts from live posts on your site, and, once inserted, will turn those posts into static content using the Header and Paragraph blocks, linking back to your site.
+
+Need to target a post title to your newsletter audience? Not a problem! You can edit these blocks, and add others, to make sure they’ speak to your readers.
+
+= Add ads! =
+
+Monetize your newsletters by including advertising. The Newspack Newsletters plugin lets you create ads, which are stored as a custom post type. Ads are automatically appended to the end of your newsletters, and can also be toggled off on a newsletter-by-newsletter basis.
+
+= Send with ease =
+
+Newspack Newsletters connects to your Mailchimp or Constant Contact accounts to send emails using the mailing lists you’ve collected there, all without leaving the WordPress Admin.
+
+Before sending your campaign, you can send test emails to one or more email addresses at a time from the newsletter editor, to make sure everything is pixel perfect before sharing with the world.
+
+= About Newspack =
+
+The Newspack Newletters plugin is part of Newspack, a suite of tools to help small to mid-sized news organizations publish and generate revenue with WordPress. Newspack is a collaborative project by WordPress.com and the Google News Initiative. You can learn more about Newspack by [visiting our website](https://newspack.pub/).
+
+
+== Installation ==
+1. Upload the Newspack Newsletters plugin to your website, and activate it.
+2. Visit the Newsletters > Settings page, or go to Newsletters > Add New to be prompted to select your service provider (MailChimp or Constant Contact). For Mailchimp, you will need an [API key](https://us1.admin.mailchimp.com/account/api/), for Constant Contact, you will need an [API Key](https://constantcontact.mashery.com/) and an [Access Token](https://constantcontact.mashery.com/io-docs).
+3. Enter your [MJML Application ID and Secret Key](https://mjml.io/api).
+4. Start creating newsletters!
+
+
+== Frequently Asked Questions ==
+
+= What newsletter providers are supported? =
+
+Mailchimp and Constant Contact are currently supported.
+
+== Screenshots ==
+
+1. Pick from one of the bundled templates to start your new newsletter, or start with a blank page. You can also create and save your own templates to create a consistent newsletter series.
+2. In the editor, you can add and remove blocks, save settings, and send yourself a test email before sending out your newsletter.
+
+== Changelog ==
```

Comment:
Is that required? If we add a changelog here, it will have to be updated on release.

Reply:
I _assumed_ it was, but I don't think it actually is either! 

When running the readme through the [validator](https://wordpress.org/plugins/developers/readme-validator/) it will show a Note (rather than a Warning) when the Changelog is missing -- we also get a note for not including a Donate link. Warnings seem to be limited to the vital things like missing the 'Contributors' or 'Tested up to'. 

It'll definitely be nice to not have to manually maintain it here -- I've removed it in dd8cd31.